tests/E2E/.docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM prestashop/base:7.2-apache
+
+WORKDIR /tmp
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install -y apt-utils
+RUN apt install -y \
+    gnupg2 \
+    curl \
+    git \
+    software-properties-common \
+    nodejs
```

Comment:
Nodejs is not needed before the new repository is declared.

Reply:
Thanks :) Nice catch